packages/config/rollup.js

```diff
@@ -144,6 +150,7 @@ export const makeRollupConfig = (iifeName) =>
     outputOptions: {
       exports: "default",
       globals: { jspsych: "jsPsychModule" },
+      plugins: [cffToJsonPlugin()],
```

Comment:
I haven't looked into the details yet, but have you tried playing with the plugin order? Briefly looking at the plugin code, I think the custom plugin should be at the end of the plugin chain.

Reply:
So this order works OK for the node.js builds when we don't have to bundle external dependencies. We thought it made sense to put it earlier because it's just doing a code injection and we thought having the stuff that comes after process that injection made sense.

Reply:
Just confirmed that moving the custom plugin to the end doesn't fix the issue.